openproblems/tasks/cell_cell_communication/README.md

```diff
@@ -0,0 +1,72 @@
+# Cell-cell Communication
+
+## The task
+
+The growing availability of single-cell data has sparked an increased
+interest in the inference of cell-cell communication (CCC),
+with an ever-growing number of computational tools developed for this purpose.
+
+Different tools propose distinct preprocessing steps with diverse
+scoring functions, that are challenging to compare and evaluate.
+Furthermore, each tool typically comes with its own set of prior knowledge.
+To harmonize these, we recently developed the
+[LIANA](https://github.com/saezlab/liana) framework, which is
+used extensively at the current stage of the task.
+
+The challenges in evaluating the tools are further exacerbated by the
+lack of an appropriate gold standard to benchmark the performance of
+CCC methods. In a [recent publication](https://rdcu.be/cR69y), we used
+alternative data modalities, including the spatial proximity of cell types
+and inferred downstream cytokine activities, to assess the ability
+of the methods to detect biological signal. Yet, these modalities are only
+approximations of biological reality and come with their own assumptions
+and limitation. We thus hope that the **OpenProblems** platform will allow
+for the inclusion of more datasets with known ground truth interactions,
+with which the limitations and advantages of the
+different CCC methods will be better understood.
+
+## The metrics
+
+Metrics for cell-cell communication aim to characterize how good are
+the different scoring functions/methods at prioritizing
+assumed truth predictions.
+
+`Odds ratios`: represent the ratio of true and false positives within a set of
+prioritized interactions (top ranked hits) versus the same ratio for the
+remainder of the interactions. Thus, in this scenario odds ratios quantify
+the strength of association between the ability of methods to prioritize
+interactions and those interactions assigned to the positive class.
+
+## API
+
+Datasets should include cell type annotations in `adata.obs["label"]`,
+along with some assumed truth in `adata.uns["bench"]` with a binary
```

Comment:
What do you think about renaming `bench` to `ccc_true` and `ccc` to `ccc_pred`?

Reply:
Initially, I had 'bench' as 'truth' but didn't want to imply actual truth, so I felt that 'bench' would be more cautious. Truth is generally unknown in the field.

I can change them to `ccc_bench` and `ccc_pred`, but would prefer to avoid true or truth :)

Reply:
Can we say `ccc_target` or something then? `bench` doesn't mean much to me semantically.

Reply:
Sounds good - these are now changed accordingly.